wavesrv/pkg/sstore/dbops.go

```diff
@@ -854,6 +854,18 @@ func GetCmdByScreenId(ctx context.Context, screenId string, lineId string) (*Cmd
 	})
 }
 
+func UpdateWithAddNewOpenAICmdInfoPacket(ctx context.Context, screenId string, pk *packet.OpenAICmdInfoChatMessage) (*ModelUpdate, error) {
+	ScreenMemAddCmdInfoChatMessage(screenId, pk)
+	cmdInfoUpdate := ScreenMemGetCmdInfoChat(screenId).Messages
+	return &ModelUpdate{OpenAICmdInfoChat: cmdInfoUpdate}, nil
+}
+
+func UpdateWithUpdateOpenAICmdInfoPacket(ctx context.Context, screenId string, messageID int, pk *packet.OpenAICmdInfoChatMessage) (*ModelUpdate, error) {
+	ScreenMemUpdateCmdInfoChatMessage(screenId, messageID, pk)
+	cmdInfoUpdate := ScreenMemGetCmdInfoChat(screenId).Messages
+	return &ModelUpdate{OpenAICmdInfoChat: cmdInfoUpdate}, nil
```

Comment:
Currently sending the entire chat history every time
This seems to work fine without any issues of latency, and I think we can probably safely have a max history length to avoid crazy long histories, but another optimization is we can send just the newest message that is actually being streamed and then have the client update its array. I'm wondering if this is needed and I'm leaning towards that it is
This method is super wasteful, but it is small amounts of data
I guess the web socket is a shared and valuable resource